lib/hq/graphql/paginated_association_loader.rb

```diff
@@ -17,6 +17,7 @@ def self.for(*args, scope: nil, **kwargs)
       end
 
       def initialize(model, association_name, internal_association: false, limit: nil, offset: nil, scope: nil, sort_by: nil, sort_order: nil)
+        super()
```

Comment:
What's going on here?

Reply:
This is the offense (the same applies for the others)
![Screenshot from 2020-08-10 18-28-14](https://user-images.githubusercontent.com/4020310/89837605-51c87f80-db37-11ea-9615-a5523a750f74.png)

Reply:
I just disabled new cops in our default rubocop config so this shouldn't be an issue anymore. Plesae remove.